tests/Services/ConvertToJsonServiceTest.php

```diff
@@ -47,4 +47,29 @@ public function testConvert_malformed_message()
         $this->expectException(TypeError::class);
         \ProjectManager\Services\ConvertToJsonService::convert([], '');
     }
+
+    public function testNoTasksAssignedToThatUserErrorResponse_success()
+    {
+        $result = \ProjectManager\Services\ConvertToJsonService::NoTasksAssignedToThatUserErrorResponse();
+        $expected = '{"message":"No tasks assigned to that user for this project","data":[]}';
+        $this->assertSame($expected, $result);
+        $this->assertSame(404, http_response_code());
+    }
+
+    public function testInvalidProjectIdResponse_success()
```

Comment:
think we need extra tests for the different pathways through the else if statements (i.e. where there is some OR logic?)